ecsact/entt/runtime.hh

```diff
@@ -1112,10 +1192,30 @@ private:
 				}
 			});
 		}
+
+		for(int i = 0; options.destroy_entities_length > i; ++i) {
+			const ecsact_entity_id& entity = options.destroy_entities[i];
+			mp_for_each<typename package::components>([&]<typename C>(C) {
+				if constexpr(C::transient) {
+					return;
+				}
+				if(has_component<C>(reg_id, entity)) {
```

Comment:
can we use the registry directly instead of `has_component`. e.g. `info.registry.all_of<C>(entt_entity_id)` would work. This would allow us to remove the `reg_id` parameter.